src/components/Table/GridTable.tsx

```diff
@@ -1044,6 +1060,12 @@ function toContent(
   }
   if (content && typeof content === "string" && isHeader && canSortColumn) {
     return <SortHeader content={content} iconOnLeft={alignment === "right"} />;
+  } else if (content && typeof content === "string" && style === beamFixedStyle) {
+    return (
+      <span css={Css.mw0.truncate.$} title={content}>
```

Comment:
Is this like `GridStyle.truncate: boolean` instead of `=== beamFixedStyle`? 

Fwiw I'm assuming that each of the flags I'm suggesting, `truncate`, `borderless`, `typeScale`, are for the most part orthogonal and so mix-and-matchable...if we have something that just can't work that way, please call it out...

Reply:
I can't think of anything off hand, so this approach may work just fine. Will definitely call it out if I think of one.